charts/ccd-message-publisher/Chart.yaml

```
@@ -2,14 +2,14 @@ apiVersion: v2
 description: A Helm chart for the HMCTS CCD Message Publisher
 name: ccd-message-publisher
 home: https://github.com/hmcts/ccd-message-publisher
-version: 0.1.11
+version: 0.1.12
 maintainers:
   - name: HMCTS CCD Dev Team
 dependencies:
   - name: java
     version: 4.0.13
     repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
   - name: servicebus
-    version: 0.4.0
+    version: 1.0.1
```

```suggestion
    version: 1.0.2
```
is released, is without the queue that was coming from those default chart values before